.github/workflows/deploy-play-store.yml

```diff
@@ -0,0 +1,68 @@
+# This is a basic workflow that is manually triggered
+
+name: Deploy to PLay Store
+
+env:
+  # The name of the main module repository
+  main_project_module: app
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "greet"
+  greet:
```

Comment:
Could you modify the name of the job to a more relevant one to the one that has been created, like deployToPlayStore?